README.md

```diff
@@ -481,8 +483,28 @@ negotiator.authenticate({
 	unsignedToken,
 });
 
-negotiator.on("proof", () => {
+negotiator.on("proof", ({ proof, issuer, error }) => {
+	// the proof will be received here (valid or failed)
+});
+```
+
+### Authenticate ownership of multiple off chain Tokens
+
+Authenticating ownership of the tokens will provide a list of proof data with a limited expiry.
+
+```javascript
+/**
+ * @param {{issuer: String, unsignedToken: Object}[]} tokens object array of tokens
+ */
+negotiator.authenticate([{
+	issuer,
+	unsignedToken,
+}]);
+
+negotiator.on("proof", ({ issuers, issuersValidated }) => {
 	// the proof will be received here (valid or failed)
+	// issuers: { issuerName: [ token, ... ] }
+	// issuersValidated: [ 'issuerName1', 'issuerName2', '...' ]
```

Comment:
@nicktaras , issuers object will containe only valid data, right? in that case all missings issuers invalid. so "issuersValidated" property contains duplicated data, do I miss something?

Reply:
I'll change this to "issuersProcessed" to make it clearer to the end developer these are the issuers that were processed as part of the authentication.

Reply:
I'll leave this conversation open to review back with you on this.

Reply:
hi, @nicktaras , can't we use Object.keys(issuers) instead of issuersProcessed/issuersValidated ?

Reply:
Yep, I'll remove this, it's not essential. I had added this initially from past experience where dev teams liked to do it like this rather than use object keys or for key in etc.